speid/validations/speid_transaction.py

```diff
@@ -52,11 +52,12 @@ def to_dict(self):
         }
 
     def __post_init__(self):
-        if len(self.cuenta_beneficiario) == 16:
-            self.tipo_cuenta_beneficiario = AccountType.DEBIT_CARD.value
+        if len(self.cuenta_beneficiario) == 18:
+            self.tipo_cuenta_beneficiario = TipoCuenta.clabe.value
+        else:
+            self.tipo_cuenta_beneficiario = TipoCuenta.card.value
```

Comment:
better to have something on these lines
```python
        cuenta_len = len(self.cuenta_beneficiario)
        if cuenta_len == 18:
            self.tipo_cuenta_beneficiario = TipoCuenta.clabe.value
        elif cuenta_len in {15, 16}:
            self.tipo_cuenta_beneficiario = TipoCuenta.card.value
        else:
            raise ValueError(f'{cuenta_len} is not a valid cuenta length')
```

Reply:
Fixed `Transaction.get_order()`. However I had to add some `if`'s since some fields are not `Optional`

Reply:
much better!